packages/dataviews/src/item-actions.js

```diff
@@ -168,10 +170,16 @@ export default function ItemActions( { item, actions, isCompact } ) {
 						icon={ moreVertical }
 						label={ __( 'Actions' ) }
 						disabled={ ! secondaryActions.length }
+						className="dataviews-view-table__all-actions-button"
```

Comment:
We shouldn't use classnames for specific views in the generic component. I have opened a follow up [PR](https://github.com/WordPress/gutenberg/pull/59748) for this.